helm/values.yaml

```diff
@@ -97,3 +97,19 @@ hpa:
   minReplicas: 1
   maxReplicas: 5
   targetCPUUtilizationPercentage: 80
+  behavior:
+    enabled: true
+    scaleUp:
+      policy:
+        type: Pods
+        value: 2
```

Comment:
shouldn't the max/min values in L105 and L112 align with existing `minReplicas` and `maxReplicas`?